### PR TITLE
fix: Fix nightly and activity failing with non ApplicationFailure

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: packages/test/src/test-worker-activities.ts [hunk 1]
```diff
@@ -253,3 +253,27 @@ test('Worker cancels activities after shutdown', async (t) => {
   t.truthy(result?.failed);
   t.true(activityCancelled);
 });
+
+test('Non ApplicationFailure TemporalFailures thrown from Activity are wrapped with ApplicationFailure', async (t) => {
```
Comment (lorensr) [inline packages/test/src/test-worker-activities.ts:257]: Should `CancelledFailure`s be wrapped in ApplicationFailures?

File: packages/worker/src/worker-options.ts [hunk 2]
```diff
@@ -93,7 +93,7 @@ export interface WorkerOptions {
    * Time to wait for pending tasks to drain after shutdown was requested.
    *
    * @format {@link https://www.npmjs.com/package/ms | ms} formatted string or number of milliseconds
-   * @default 5s
+   * @default 10s
```
Comment (lorensr) [inline packages/worker/src/worker-options.ts:96]: I don't see this changed in the code—was the number just wrong before?

**Review by lorensr — APPROVED** [review]
I think wrapping more things in ApplicationFailures is potentially breaking, for those who do a specific number of `.cause`s in their error checks.

File: packages/test/src/test-worker-activities.ts (hunk 1 above)
Comment (bergundy) [reply packages/test/src/test-worker-activities.ts:257]: Yes, unless the activity was requested to be cancelled

File: packages/worker/src/worker-options.ts (hunk 2 above)
Comment (bergundy) [reply packages/worker/src/worker-options.ts:96]: 🤦‍♂️

**Comment by bergundy** [comment]
@lorensr we’re not wrapping more things, we just ensure that only ApplicationFailure can be the failure returned from an activity. The server expects that

**Pull request merged**